src/mono/nuget/Microsoft.NET.Sdk.Mono.Toolchain.Manifest/WorkloadManifest.targets

```
@@ -0,0 +1,44 @@
+<Project>
+    <PropertyGroup Condition="'$(UsingMicrosoftNETSdkBlazorWebAssembly)'=='true' AND '$(RunAOTCompilation)' == 'true'">
+      <UsingBlazorAOTWorkloadManifest>true</UsingBlazorAOTWorkloadManifest>
+    </PropertyGroup>
+
+    <Import Condition="'%(RunAOTCompilation)' == 'true'" Project="Sdk.props" Sdk="Microsoft.NET.Runtime.MonoAOTCompiler.Task" />
+
+    <ImportGroup Condition="'$(TargetPlatformIdentifier)' == 'android' and '$(RunAOTCompilation)' == 'true'">
+        <Import Project="Sdk.props" Sdk="Microsoft.NetCore.App.Runtime.AOT.Cross.android-x86" />
+        <Import Project="Sdk.props" Sdk="Microsoft.NetCore.App.Runtime.AOT.Cross.android-x64" />
+        <Import Project="Sdk.props" Sdk="Microsoft.NetCore.App.Runtime.AOT.Cross.android-arm" />
+        <Import Project="Sdk.props" Sdk="Microsoft.NetCore.App.Runtime.AOT.Cross.android-arm64" />
+    </ImportGroup>
+
+    <ImportGroup Condition="'$(TargetPlatformIdentifier)' == 'ios' and '$(RunAOTCompilation)' == 'true'">
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm" />
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm64" />
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.ios-x86" />
+    </ImportGroup>
+
+    <ImportGroup Condition="'$(TargetPlatformIdentifier)' == 'tvos' and '$(RunAOTCompilation)' == 'true'">
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.tvos-arm64" />
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.tvos-x64" />
+    </ImportGroup>
+
+    <ImportGroup Condition="'$(TargetPlatformIdentifier)' == 'iossimulator' and '$(RunAOTCompilation)' == 'true'">
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.iossimulator-arm64" />
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.iossimulator-x64" />
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.ios-x86" />
+    </ImportGroup>
+
+    <ImportGroup Condition="'$(TargetPlatformIdentifier)' == 'tvossimulator' and '$(RunAOTCompilation)' == 'true'">
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.tvossimulator-arm64" />
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.tvossimulator-x64" />
+    </ImportGroup>
+
+    <ImportGroup Condition="'$(UsingMicrosoftNETSdkBlazorWebAssembly)'=='true' AND '$(RunAOTCompilation)' == 'true'">
```

I think we can do something like
```suggestion
    <ImportGroup Condition="'$(TargetPlatformIdentifier)' == 'browser' AND '$(RunAOTCompilation)' == 'true'">
```
Without changing any of the current logic. I'd Also like to switch these conditions to use WasmNativeWorkload rather than RunAOTCompilation but I'll write that up separately since it will impact the Sdk